docs/building-whisper-cuda-windows.md

```
@@ -0,0 +1,112 @@
+node packages\whisper-wrapper\scripts\apply-patches.js
+pnpm --filter @amical/whisper-wrapper build
+
+:: 4. Build the CUDA variant.
+pushd packages\whisper-wrapper
+pnpm run build:native:cuda
+popd
+```
+
+If everything went well you end up with:
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)